src/main/java/core/basesyntax/service/activity/ActivityHandler.java

```diff
@@ -0,0 +1,8 @@
+package core.basesyntax.service.activity;
+
+import core.basesyntax.model.Activity;
+
+public interface ActivityHandler {
+
+    void processActivity(Activity activity);
```

Comment:
I think handlers need to take two int parameters, and only subtract or sum values